bedrock/contentful/api.py

```diff
@@ -257,6 +257,8 @@ def render(self, node):
         data_cta = ""
 
         # add referral info to links to other mozilla properties
+        # TODO: can we make this just if url.netloc == "mozilla.org" or are we expecting
```

Comment:
Right. This block is for linking to other mozilla.org subdomains, and the one below is for other non mozilla.org domains. We could perhaps update this to deal with `*.firefox.com` domains as well.

Reply:
@pmac can you elaborate on this (DMs are fine) - i'll add the changes to the next block of contentful tests